corehq/apps/api/resources/v0_5.py

```diff
@@ -326,6 +331,40 @@ def _update(cls, bundle, user_change_logger=None):
 
         return errors
 
+    def prepend_urls(self):
+        return [
+            url(r"^(?P<resource_name>%s)/(?P<pk>\w[\w/-]*)/activate/$" % self._meta.resource_name,
+                self.wrap_view('activate_user'), name="api_activate_user"),
+            url(r"^(?P<resource_name>%s)/(?P<pk>\w[\w/-]*)/deactivate/$" % self._meta.resource_name,
+                self.wrap_view('deactivate_user'), name="api_deactivate_user"),
+        ]
+
+    @location_safe
+    def activate_user(self, request, **kwargs):
+        return self._modify_user_status(request, **kwargs, active=True)
+
+    @location_safe
+    def deactivate_user(self, request, **kwargs):
+        return self._modify_user_status(request, **kwargs, active=False)
+
+    def _modify_user_status(self, request, active, **kwargs):
+        self.is_authenticated(request)
+
+        try:
+            user = CommCareUser.get(kwargs['pk'])
+            assert user.domain == kwargs['domain']
+            verify_modify_user_conditions(request, user, active)
+        except ModifyUserStatusException as e:
+            raise BadRequest(_(str(e)))
+
+        user.is_active = active
+        user.save(spawn_task=True)
+        log_user_change(by_domain=request.domain, for_domain=user.domain,
+                        couch_user=user, changed_by_user=request.couch_user,
+                        changed_via=USER_CHANGE_VIA_API, fields_changed={'is_active': user.is_active})
+
+        return self.create_response(request, {}, response_class=http.HttpAccepted)
```

Comment:
Log throttle access should be called for successful requests:

```suggestion
        self.log_throttled_access(request)
        return self.create_response(request, {}, response_class=http.HttpAccepted)
```